abci/example/kvstore python3/Readme.md

```diff
@@ -0,0 +1,28 @@
+# KVStoreApplication
+
+The KVStoreApplication is a simple merkle key-value store. Transactions of the form key=value are stored as key-value pairs in the tree. Transactions without an = sign set the value to the key. The app has no replay protection (other than what the mempool provides).
```

Comment:
> Transactions without an = sign set the value to the key

what key?

Reply:
I just copy pasted from kvstore docs 😅

Reply:
Is the logic of the kv store application code right ? Is this how you write a Tendermint application 😅 I’m still very unsure.

Reply:
I personally think you shouldn't store db in a state, but rather have application to store a link to it.

```
self.db = NewMemDB()
self.state = State.load_state(self.db)
```

Reply:
> I personally think you shouldn't store db in a state, but rather have application to store a link to it.
That's what is happening for now.

About implementing the persistant kvstore, is it fine if i use sqlite3 instead of leveldb. https://github.com/davebryson/py-tendermint/blob/master/tendermint/db.py 

>  Using sqlite3 for key/value store...what!?  Here's why:  existing tech (leveldb, codernitydb, etc...) either doesn't  support windows, not ported to Python 3, or both.

I haven't looked into this problem yet, so i'm not sure.